### PR TITLE
secure route checking moved to ods-contrib

### PR DESCRIPTION
**Pull request by henrjk**
Doc Update related to https://github.com/opendevstack/ods-core/pull/449

Is there a version 2.1.x?

### CODE REVIEW

File: docs/modules/administrator-guides/pages/update.adoc
Comment (gerardcl) [inline docs/modules/administrator-guides/pages/update.adoc:109]:
```suggestion
This secure route checking code has been removed in ODS version 2.x as this is an
```

**Comment by gerardcl** [comment]
Hi @henrjk is `ods-contrib` planned to exist out of this github project, like to be in our bix-digital github project only?

File: docs/modules/administrator-guides/pages/update.adoc
Comment (michaelsauter) [inline docs/modules/administrator-guides/pages/update.adoc:109]: This should not be removed in 2. Only on master, which next release will be 3.

**Review by michaelsauter — CHANGES_REQUESTED** [review]

**Comment by henrjk** [comment]
> Hi @henrjk is `ods-contrib` planned to exist out of this github project, like to be in our bix-digital github project only?

At the moment there was no plan to have it on GitHub also.

**Comment by michaelsauter** [comment]
BTW, there will not be a `2.1.x`.

There is:
`master` (branch)
`2.x` (branch; and there will be a `3.x`)
`v2.0` (tag; and there will be a `v2.1`)

File: docs/modules/administrator-guides/pages/update.adoc
Comment (henrjk) [reply docs/modules/administrator-guides/pages/update.adoc:109]: Should the note say that it has been removed on master and will be removed on the next release or should we wait until release time and then have the commit ready.

**Comment by michaelsauter** [comment]
Given https://github.com/opendevstack/ods-documentation/issues/42, it would be best to hold off with this, and I'll merge the changes here in once that is merged.

**Comment by michaelsauter** [comment]
Applied in https://github.com/opendevstack/ods-core/commit/f7ed8e05ca792a019a0ad432520bac35431551e9.

**Pull request closed without merging**
